_maps/RandomRuins/IceRuins/icemoon_underground_wrath.dmm

```diff
@@ -0,0 +1,267 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"b" = (
+/turf/closed/mineral/random/snow/no_caves,
+/area/icemoon/underground/explored)
+"e" = (
+/obj/structure/spawner/nether,
```

Comment:
If you'd like a loot idea for this wrath ruin, perhaps a pair of gloves that let you rip apart (or butcher, for simplemobs) corpses with your bare hands? Because you hate hate hate HATE HATE HATE THEM SO MUCH DIE DIE DIE AND NEVER COME BACK I'LL RIP YOU TO ATOMS SO I NEVER HAVE TO SEE YOUR SMUG FACE AGAIN YOU PIECE OF SHIT RAAAAAAAAAAAAAAAAAAAAAAAGH

Reply:
oh wow that's actually way better than the trash idea i was planning thanks

cool miner gear like that is the best